### ISSUE TITLE
Translation Error: "No" is translated to "Yes" in Korean

### ISSUE THREAD

**Issue by ManwooJ** [MicrosoftDocs/azure-docs.ko-kr#1132]
[여기에 피드백 입력]
For Micorsoft.LabSerivces, "No" is translated to "예" (Yes).

This is causing confusion because it introduces Korean users unsupported operations as supported operations.
Me and my Korean customer read the page in English and Korean respectively and understood it differently.

The following is the one we discussed today.

![image](https://user-images.githubusercontent.com/31500252/96947981-85ecf300-151f-11eb-83dd-2f56692d6ef5.png)

![image](https://user-images.githubusercontent.com/31500252/96948082-c0569000-151f-11eb-9c6e-f961980dfb51.png)

Note: I didn't check if other resources are okay, because too many resources are listed in this page.

---
#### 문서 세부 정보

⚠ *이 섹션을 편집하지 마세요. docs.microsoft.com ➟ GitHub 문제 연결에 필요합니다.*

* ID: 23df5354-6833-a5da-0eb3-d860150c8b78
* Version Independent ID: 682ec206-0fda-3e0b-5841-543860a87e34
* Content: [리소스 종류별 이동 작업 지원 - Azure Resource Manager](https://docs.microsoft.com/ko-kr/azure/azure-resource-manager/management/move-support-resources#feedback)
* Content Source: [articles/azure-resource-manager/management/move-support-resources.md](https://github.com/MicrosoftDocs/azure-docs.ko-kr/blob/master/articles/azure-resource-manager/management/move-support-resources.md)
* Service: **azure-resource-manager**
* Sub-service: **management**
* GitHub Login: @tfitzmac
* Microsoft Alias: **tomfitz**

**Comment by WooSung-Jung** [comment]
I agree this issue. Please you should Pull request this issue of file. If you can't this. You write comment to request me. Best regard.

**Comment by WooSung-Jung** [comment]
This issue not updated recent. I will update Pull request soon.

**Comment by WooSung-Jung** [comment]
Should you close this issue.

**Comment by ManwooJ** [comment]
The doc is still incorrect. 
I checked the date on the top of the doc, and it was 2021.01.11.
If it hasn't been updated yet, when is the fix applied? 

p.s. I'm sorry for forgetting this so long, and thank you for pull request.

**Comment by WooSung-Jung** [comment]
@ManwooJ
Unfortunately contributor can't deploy.
Deployment will take little time.
Please wait.

**Comment by ManwooJ** [comment]
@WooSung-Jung yes, of course :) I read the discussion in your pull request, and just wanted to know if you knew the timeline.
I'll close this issue. Thank you!

**Comment by WooSung-Jung** [comment]
@ManwooJ
@olprod 

I check commit history. but LabSerivces don't changed.

> https://github.com/MicrosoftDocs/azure-docs.ko-kr/commit/f6e63dee4fad8284be3c552c2809f78f3adfb2cb#diff-f317e4221f20474ca33d02e0ae3f8bb0ee1a144a76d769a659bf8ab5395ba0ce

**Comment by ManwooJ** [comment]
@WooSung-Jung @olprod 

Still not fixed. Could you check this issue again?

**Comment by WooSung-Jung** [comment]
@ManwooJ @olprod 

I uploaded full request at 18days ago. But repository admin has not yet merged.

**Comment by WooSung-Jung** [comment]
@ManwooJ 
Please issue close.

https://github.com/MicrosoftDocs/azure-docs.ko-kr/pull/1151

**Comment by ManwooJ** [comment]
Hi,

These days I have an issue with login to GitHub, so I can't do anything.
But the thing is the Korean page is still not fixed.
I don't understand why your correct doesn't work, but please check it again.

Manwoo
________________________________

**Comment by Arc-Jung** [comment]
@ManwooJ
Please issue close.
